mmdet/models/dense_heads/anchor_head.py

```diff
@@ -691,13 +691,13 @@ def _get_bboxes(self,
         if deploy_nms_pre > 0 and torch.onnx.is_in_onnx_export():
             # Get maximum scores for foreground classes.
             if self.use_sigmoid_cls:
-                batch_mlvl_scores, _ = batch_mlvl_scores.max(-1)
```

Comment:
Is the rename necessary?

Reply:
Yes, this may not match the title. This fix is ​​a bug in the PR [Refactor] batch infer, which is the previous merge, and it will only occur during onnx.export; since there was no unit test related to onnxruntime in the ci, this bug could not be found.